src/main/scala/org/neo4j/spark/Neo4jConfig.scala

```diff
@@ -1,34 +1,41 @@
 package org.neo4j.spark
 
 import org.apache.spark.SparkConf
-import org.neo4j.driver.v1.{Driver, AuthTokens, Config, GraphDatabase}
+import org.neo4j.driver.{AuthToken, AuthTokens, Config, Driver, GraphDatabase, Session, SessionConfig}
 
 /**
  * @author mh
  * @since 02.03.16
  */
-case class Neo4jConfig(val url: String, val user: String = "neo4j", val password: Option[String] = None, encryptionStatus:Boolean) {
```

Comment:
should we also add support for multi-db here? specify the db to connect to?

Reply:
There is the multi support db via config property `spark.neo4j.database`.